docs/source/get_started/model_data/model_data.rst

```diff
@@ -2595,6 +2595,84 @@ copy migration files from installed modules to the :code:`.motech` directory. Fi
 directory(if you are using mysql then use :code:`mysql` instead :code:`default`) in the bundle. Each file muse have
 `a proper name <http://flywaydb.org/documentation/migration/sql.html>`_(e.g. :code:`V1__Description.sql`).
 
+####################
+Schema Import/Export
+####################
+
+This section will explain how to import/export MDS Schema. It is fairly easy and can be done with only few steps using
+the MDS module UI.
+
+Please go to the following subsections to learn how to do that
+
+    - `Schema Import`_
+    - `Schema Export`_
```

Comment:
You can place those links in the first sentence of this section, where you say this document covers MDS import and export. Just make import and export there a link.